generators/ci-cd/command.ts

```diff
@@ -41,13 +49,16 @@ const command: JHipsterCommandDefinition = {
         { name: 'Travis CI', value: 'travis' },
         { name: 'CircleCI', value: 'circle' },
       ],
-      scope: 'context',
+      scope: 'storage',
```

Comment:
Why did the scope value change?

Reply:
In order to persist the choices of the end user. I do not see any useful usage of 'context' TBH